datacontract/engines/soda/connections/sqlserver.py

```diff
@@ -0,0 +1,41 @@
+import os
+
+import yaml
+from datacontract.model.data_contract_specification import Server
+
+def to_sqlserver_soda_configuration(server: Server) -> str:
+    """Serialize server config to soda configuration.
+
+
+    ### Example:
+        type: sqlserver
+        host: host
+        port: '1433'
+        username: simple
+        password: simple_pass
+        database: database
+        schema: dbo
+        trusted_connection: false
+        encrypt: false
+        trust_server_certificate: false
+        driver: ODBC Driver 18 for SQL Server
+  """
+    # with service account key, using an external json file
+    soda_configuration = {
+        f"data_source {server.type}": {
+            "type": "sqlserver",
+            "host": server.host,
+            "port": str(server.port),
+            "username": os.getenv("DATACONTRACT_SQLSERVER_USERNAME", ''),
+            "password": os.getenv("DATACONTRACT_SQLSERVER_PASSWORD", ''),
+            "database": server.database,
+            "schema": server.schema_,
+            "trusted_connection": os.getenv("DATACONTRACT_TRUSTED_CONNECTION", False),
+            "trust_server_certificate": os.getenv("DATACONTRACT_TRUST_SERVER_CERTIFICATE", False),
+            "encrypt":  os.getenv("DATACONTRACT_ENCRYPTED_CONNECTION", True),
```

Comment:
I'd prefix the other settings to DATACONTRACT_SQLSERVER_ as well

Reply:
Fair enough; I'll make them more consistent